### ISSUE TITLE
安装成功但是 训练的时候出错 

### ISSUE THREAD

**Issue by YourUncleKong** [Akegarasu/lora-scripts#75]
安装成功了 但是想要训练的时候 出现了这个error 是什么原因啊大大 我是跟着你的视频一步一步做的 前面都没问题                                                  

Fatal error in launcher: Unable to create process using '"E:\AI\Lora-train\venv\Scripts\python.exe"  "E:\AI\Lora-train_aki\venv\Scripts\accelerate.exe" launch --num_cpu_threads_per_process=8 ./sd-scripts/train_network.py --enable_bucket --pretrained_model_name_or_path=./sd-models/model.ckpt --train_data_dir=./train/Character_Fat_out --output_dir=./output --logging_dir=./logs --resolution=512,512 --network_module=networks.lora --max_train_epochs=10 --learning_rate=1e-4 --unet_lr=1e-4 --text_encoder_lr=1e-5 --lr_scheduler=cosine_with_restarts --lr_warmup_steps=0 --network_dim=32 --network_alpha=32 --output_name=Character_Fat_out --train_batch_size=1 --save_every_n_epochs=2 --mixed_precision=fp16 --save_precision=fp16 --seed=1337 --cache_latents --clip_skip=2 --prior_loss_weight=1 --max_token_length=225 --caption_extension=.txt --save_model_as=safetensors --min_bucket_reso=256 --max_bucket_reso=1024': The system cannot find the file specified.

Train finished

![image](https://user-images.githubusercontent.com/130347671/230852079-34882739-3aa4-4c70-b594-7f26e13a5645.png)

**Comment by 4nth0ny1130** [comment]
看起来是python解释器路径错了,如果你的venv是python -m venv命令创建的那么venv路径下一般没有python解释器

**Comment by 4nth0ny1130** [comment]
看起来是python解释器路径错了,如果你的venv是`python -m venv venv`命令创建的那么venv路径下不会有python解释器

**Comment by 4nth0ny1130** [comment]
看起来是python解释器路径错了,如果你的`E:\AI\Lora-train\venv`文件夹是`python -m venv venv`命令创建的那么venv路径下不会有python解释器

**Comment by HouseYeung** [comment]
怀疑你安装结束后更改了整个程序文件夹的路径，不知道更改路径后怎么设置才能让它继续运行
